### PR TITLE
Changed the hardcoded redis-keys to CPP constants

### LINKED ISSUE #2728: Update UI board display code to use software constants
### Description of the task
Currently ui board display code is using hard coded redis keys. These should be updated to be using the constants from `software/constants.h`. This can be done by importing software.py_constants and replacing the existing strings

Requires a jetson and old UI board + display to test - potentially can get myself or someone else to test it for you

### Acceptance criteria

- [ ] no more hardcoded keys in software/jetson_nano/display  

### Blocked By

https://github.com/UBC-Thunderbots/Software/pull/2721

### PR DESCRIPTION
**Pull request by rachad47**
### Description@@ -32,11 +33,11 @@ class ScreenActions:

   Went to existing ,py files that were using the constants from the py_constants.cpp and checked the way they imported the C++ file into python so I can the same under the .py file I edited.

Resolves #2728

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/constants.h [hunk 1]
```diff
@@ -49,6 +49,7 @@ const std::string ROBOT_KICK_CONSTANT_REDIS_KEY     = "/kick_constant";
 const std::string ROBOT_CHIP_PULSE_WIDTH_REDIS_KEY  = "/chip_pulse_width";
 const std::string ROBOT_CURRENT_DRAW_REDIS_KEY      = "/current_draw";
 const std::string ROBOT_BATTERY_VOLTAGE_REDIS_KEY   = "/battery_voltage";
+const std::string CAPACITOR_VOLTAGE_REDIS_KEY       = "/cap_voltage";
```
Comment (LiCody) [inline src/software/constants.h]:
```suggestion
const std::string ROBOT_CAPACITOR_VOLTAGE_REDIS_KEY = "/cap_voltage";
```
Can we change this to match the other variable names in the file

**Review by LiCody — CHANGES_REQUESTED** [review]
Also some occurrences in /software/jetson_nano/display/screens/home_screen.py

File: src/software/jetson_nano/display/robot_ui.py [hunk 2]
```diff
@@ -32,11 +33,11 @@ class ScreenActions:
 # These are the keys for the redis dicationary
 # TODO(#2728): Replace key strings with values from constants
```
Comment (nimazareian) [inline src/software/jetson_nano/display/robot_ui.py]: Now that you're resolving a ticket, don't forget to remove it's TODOs in the code as well.
You can search globally in CLion (`ctrl + shift + f`) or use `ag` in the terminal from the `src` folder to search the ticket number (i.e. `ag 2728`), and find the locations of the two other TODO comments for this ticket as well.

Comment (nimazareian) [reply src/software/jetson_nano/display/robot_ui.py]: The two other TODOs will also show you some of the other places where the hard coded string keys can be replaced with these constants. One of them is in `home_screen.py`. The other TODO in `network.cpp` I think is actually labelled incorrectly and should be tagging #2727 instead. I would appreciate it if you could update that as well!

**Review by nimazareian — CHANGES_REQUESTED** [review]
Overall looks good to me, just had 1 nit and noticed that a few variables still need to be updated.

**Review by LiCody — APPROVED** [review]

**Review by nimazareian — APPROVED** [review]
LGTM! 💯

**Pull request merged**